src/test/java/steps/HomePageTestSteps.java

```diff
@@ -0,0 +1,25 @@
+package steps;
+
+import base.TestBase;
+import pages.CatalogPage;
+import pages.ElectronicsPage;
+import pages.HomePage;
+import pages.Smartphone;
+
+public class HomePageTestSteps extends TestBase {
+    HomePage homePage = new HomePage();
+    ElectronicsPage electronicsPage = new ElectronicsPage();
+    CatalogPage catalogPage = new CatalogPage();
+    Smartphone smartphone = new Smartphone();
```

Comment:
I think POs variables can be stored in TestBase because they will be used in each test class, do you agree?

Reply:
just noticed, why steps extend TestBase? doesn't  TestBase relate to tests?